index.html

```diff
@@ -1,20 +1,27 @@
 <!DOCTYPE html>
 <html lang="pl">
-<head>
-    <meta charset="UTF-8"/>
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
-    <meta http-equiv="X-UA-Compatible" content="ie=edge"/>
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>CodersCamp2020 | Star Wars API QUIZ</title>
-    <link rel="stylesheet" href="styles/App.css"/>
+    <link rel="stylesheet" href="styles/App.css" />
+    <link rel="stylesheet" href="styles/MainMenu.css" />
```

Comment:
Tego nie jestem pewna, dobrze jakby Mateusz doradzil, wydaje mi sie, ze style mozna w modulowym JS zaciagac bezposrednio do pilkow JS. Uzywa sie w samym pliku JS MainMenu.js na samej gorze importu. Mozesz spojrzec do pliku index.js i tam jest `import '../styles/App.css';` tak samo mozna importowac chyba style do kazdego pliku.

Reply:
Jeśli template zostanie przeniesiony do JSa to chcemy importować style `per-moduł`. 
W innym wypadku można by wykorzystać preprocesor SCSS i później wszystko lądowało by w jednym pliku ale nie byłoby potencjalnych konfliktów nazw.

Reply:
Template przeniosę do JS ale mimo wszystko może warto zainstalować SASS i importować wszystkie style w 1 plik?